sigridci/sigridci/repository_history_exporter.py

```diff
@@ -12,13 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import hashlib
```

Comment:
I think using hashing is perfectly OK. Normally, hashing is insufficient: someone who knows the set of all possible values being hashed (e.g., a list of employees) could just hash that list and compare the results with our anonymized values. However, nowhere in Sigrid do we display the resulting hashes, they're only used internally for grouping. That way neither us nor our users can circumvent this process.